R/buildRecord.R

```diff
@@ -153,29 +153,37 @@ getAnalyticalInfo <- function(cpd = NULL)
 	ac_lc[['FLOW_GRADIENT']] <- getOption("RMassBank")$annotations$lc_gradient
 	ac_lc[['FLOW_RATE']] <- getOption("RMassBank")$annotations$lc_flow
 	ac_lc[['RETENTION_TIME']] <- sprintf("%.3f min", rt)  
-	ac_lc[['SOLVENT A']] <- getOption("RMassBank")$annotations$lc_solvent_a
-	ac_lc[['SOLVENT B']] <- getOption("RMassBank")$annotations$lc_solvent_b
+	lc_solvents <- getOption("RMassBank")$annotations$lc_solvents
+	ac_lc[['SOLVENT A']] <- lc_solvents$lc_solvent_a
+	ac_lc[['SOLVENT B']] <- lc_solvents$lc_solvent_b
+	if(length(lc_solvents) > 2)
+		ac_lc[['SOLVENT C']] <- lc_solvents$lc_solvent_c
```

Comment:
What about creating the names through 
`sapply(seq(65,70, along.with=lc_solvents), function(x) paste("SOLVENT", rawToChar(as.raw(x))))`

Reply:
I‘d use this if I had to specify `SOLVENT A` - `SOLVENT Z` but for three values maximum I think it just makes the code harder to read. Thanks for the hint with `rawToChar(as.raw(x))` anyway. This will come in handy somewhere else eventually.